rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerImplTest.java

```diff
@@ -509,6 +515,48 @@ void logPegoutConfirmed() {
         assertEvent(eventLogs, 0, BridgeEvents.PEGOUT_CONFIRMED.getEvent(), new Object[]{btcTx.getHash().getBytes()}, new Object[]{pegoutCreationRskBlockNumber});
     }
 
+    private static Stream<Arguments> logPegoutTransactionCreatedValidArgProvider() {
+        List<Arguments> args = new ArrayList<>();
+
+        args.add(Arguments.of(Sha256Hash.ZERO_HASH, Collections.singletonList(Coin.SATOSHI)));
+        args.add(Arguments.of(BitcoinTestUtils.createHash(5), coinListOf(500_000, 400_000, 1_000)));
+        args.add(Arguments.of(BitcoinTestUtils.createHash(10), Collections.singletonList(Coin.ZERO)));
+        args.add(Arguments.of(BitcoinTestUtils.createHash(10), Collections.EMPTY_LIST));
+        args.add(Arguments.of(BitcoinTestUtils.createHash(15), null));
+
+        return args.stream();
+    }
+
+    @ParameterizedTest()
+    @MethodSource("logPegoutTransactionCreatedValidArgProvider")
+    void logPegoutTransactionCreated_ok(Sha256Hash btcTxHash, List<Coin> outpointValues) {
+        eventLogger.logPegoutTransactionCreated(btcTxHash, outpointValues);
+        commonAssertLogs(eventLogs);
+
+        assertTopics(2, eventLogs);
+        assertEvent(eventLogs, 0, BridgeEvents.PEGOUT_TRANSACTION_CREATED.getEvent(), new Object[]{btcTxHash.getBytes()}, new Object[]{
```

Comment:
```
eventIndex = 0;
event = BridgeEvents.PEGOUT_TRANSACTION_CREATED.getEvent();
topics = new Object[]{btcTxHash.getBytes()};
data = new Object[]{UtxoUtils.encodeOutpointValues(outpointValues)};
```

use meaningful names

Reply:
I agree. Putting these values in variables first will make it easier to read.

Reply:
Done